docs/src/main/sphinx/release/release-365.md

```diff
@@ -0,0 +1,144 @@
+* Fix incorrect results when casting `bigint` values to `varchar(n)` type. ({issue}`552`)
+* Fix query failure when `PREPARE` statement is used with `DROP` or `INSERT` and table or schema name 
+  contains special characters. ({issue}`9822`)
+* Fix minor memory leak when queries are abandoned during the initial query submission phase. ({issue}`9962`)
+
+## Security
+
+* Allow configuring HTTP proxy for OAuth2 authentication. ({issue}`9920`, {issue}`10069`)
+* Add group-based and owner-based query access rules to file based system access control. ({issue}`9811`)
+* Use internal names for discovery client when automatic TLS is enabled for internal communications. This allows `discovery.uri` 
+  to be configured using a normal DNS name like `https://coordinator.trino`, and still use automatic TLS certificates. ({issue}`9821`)
+* Use Kerberos operating system ticket cache if keytab file is not provided to JDBC and CLI for Kerberos authentication. ({issue}`8987`)
+* Fix internal communication automatic TLS on Java 17. ({issue}`9821`)
+
+## CLI
+
+* Use HTTPS when port is set to 443. ({issue}`8798`)
+
+## BigQuery connector
+
+* Support reading `bignumeric` type whose precision is less than or equal to 38. ({issue}`9882`)
+* Fix failure when a schema is dropped while listing tables. ({issue}`9954`)
+
+## Cassandra connector
+
+* Support reading user defined types in Cassandra. ({issue}`147`)
+
+## ClickHouse connector
+
+* Add support for [truncating tables](/sql/truncate). ({issue}`8921`)
+* Fix incorrect query results when query contains predicates on `real` type columns. ({issue}`9998`)
+
+## Druid connector
+
+* Add support for [truncating tables](/sql/truncate). ({issue}`8921`)
+
+## Elasticsearch connector
+
+* Add support for additional Elastic Cloud node roles. ({issue}`9892`)
+* Fix failure when empty values exist in numeric fields. ({issue}`9939`)
+
+## Hive connector
+
+* Allow reading empty files. ({issue}`9929`)
+* Improve performance of decimal ``avg`` aggregation. ({issue}`9738`)
+* Improve query performance over Parquet data with timestamp encoded using `int64` representation in Parquet. ({issue}`9414`)
+* Improve dynamic partition pruning efficiency. ({issue}`9866`, {issue}`9869`)
+* Improve query performance on partitioned tables or tables with small files. ({issue}`9979`)
+* Fix reporting of number of read bytes for tables using ``ORC`` file format. ({issue}`10048`)
+* Account memory used for tracking information about deleted rows when reading from Hive ACID tables. ({issue}`9914`, {issue}`10070`)
+* Fix `REVOKE GRANT OPTION` to revoke only the granting instead of revoking entire privilege. ({issue}`10094`)
+* Fix bug that skips writing bloom filters for ORC files. ({issue}`9792`)
+* Fix bug where incorrect rows were deleted when deleting from a transactional table that has original files (before the first major compaction). ({issue}`10095`)
```

Comment:
This sounds serious. Should we put it to the top and some emphasise it?

Reply:
We normally list changes in order of:
* new features
* performance improvements
* bug fixes

Not sure where we would put this to make it more prominent.